slsDetectorSoftware/src/Module.cpp

```diff
@@ -2066,29 +2032,22 @@ void Module::setBurstMode(slsDetectorDefs::burstMode value) {
 }
 
 bool Module::getCurrentSource() {
-    int retval = -1;
-    sendToDetector(F_GET_CURRENT_SOURCE, nullptr, retval);
-    LOG(logDEBUG1) << "Current source enable:" << retval;
-    return static_cast<bool>(retval);
+    return sendToDetector<int>(F_GET_CURRENT_SOURCE);
 }
 
 void Module::setCurrentSource(bool value) {
-    int arg = static_cast<int>(value);
-    LOG(logDEBUG1) << "Setting current source enable to " << arg;
-    sendToDetector(F_SET_CURRENT_SOURCE, arg, nullptr);
+    LOG(logDEBUG1) << "Setting current source enable to " << value;
```

Comment:
Do we need this printout? since you already have it in sendtoDetector right?

Reply:
done